Exception raised on the reactor's thread 'on_tunnel_remove() takes exactly 5 arguments (6 given)'.

```
I: test_on_tunnel_remove_NoneType (Tribler.Test.Community.Multichain.test_multichain_community.TestMultiChainCommunity) ... 2016-07-04 16:17:38,803 [ERROR] Exception raised on the reactor's thread 'on_tunnel_remove() takes exactly 5 arguments (6 given)'.
I:  Traceback from this thread:
I:   File "Triblerd.py", line 37, in <module>
I:     Triblerd().test()
I:   File "Triblerd.py", line 33, in test
I:     nose.run(argv=['--nocapture', '--nologcapture', '--verbose', '--stop'])
I:   File "/data/user/0/org.tribler.android/files/lib/python2.7/site-packages/nose/core.py", line 301, in run
I:     return TestProgram(*arg, **kw).success
I:   File "/data/user/0/org.tribler.android/files/lib/python2.7/site-packages/nose/core.py", line 121, in __init__
I:     **extra_args)
I:   File "/data/user/0/org.tribler.android/files/lib/python27.zip/unittest/main.py", line 95, in __init__
I:     self.runTests()
I:   File "/data/user/0/org.tribler.android/files/lib/python2.7/site-packages/nose/core.py", line 207, in runTests
I:     result = self.testRunner.run(self.test)
I:   File "/data/user/0/org.tribler.android/files/lib/python2.7/site-packages/nose/core.py", line 62, in run
I:     test(result)
I:   File "/data/user/0/org.tribler.android/files/lib/python2.7/site-packages/nose/suite.py", line 177, in __call__
I:     return self.run(*arg, **kw)
I:   File "/data/user/0/org.tribler.android/files/lib/python2.7/site-packages/nose/suite.py", line 224, in run
I:     test(orig)
I:   File "/data/user/0/org.tribler.android/files/lib/python2.7/site-packages/nose/suite.py", line 177, in __call__
I:     return self.run(*arg, **kw)
I:   File "/data/user/0/org.tribler.android/files/lib/python2.7/site-packages/nose/suite.py", line 224, in run
I:     test(orig)
I:   File "/data/user/0/org.tribler.android/files/lib/python2.7/site-packages/nose/suite.py", line 177, in __call__
I:     return self.run(*arg, **kw)
I:   File "/data/user/0/org.tribler.android/files/lib/python2.7/site-packages/nose/suite.py", line 224, in run
I:     test(orig)
I:   File "/data/user/0/org.tribler.android/files/lib/python2.7/site-packages/nose/suite.py", line 177, in __call__
I:     return self.run(*arg, **kw)
I:   File "/data/user/0/org.tribler.android/files/lib/python2.7/site-packages/nose/suite.py", line 224, in run
I:     test(orig)
I:   File "/data/user/0/org.tribler.android/files/lib/python2.7/site-packages/nose/suite.py", line 177, in __call__
I:     return self.run(*arg, **kw)
I:   File "/data/user/0/org.tribler.android/files/lib/python2.7/site-packages/nose/suite.py", line 224, in run
I:     test(orig)
I:   File "/data/user/0/org.tribler.android/files/lib/python2.7/site-packages/nose/suite.py", line 177, in __call__
I:     return self.run(*arg, **kw)
I:   File "/data/user/0/org.tribler.android/files/lib/python2.7/site-packages/nose/suite.py", line 224, in run
I:     test(orig)
I:   File "/data/user/0/org.tribler.android/files/lib/python2.7/site-packages/nose/case.py", line 45, in __call__
I:     return self.run(*arg, **kwarg)
I:   File "/data/user/0/org.tribler.android/files/lib/python2.7/site-packages/nose/case.py", line 133, in run
I:     self.runTest(result)
I:   File "/data/user/0/org.tribler.android/files/lib/python2.7/site-packages/nose/case.py", line 151, in runTest
I:     test(result)
I:   File "/data/user/0/org.tribler.android/files/lib/python27.zip/unittest/case.py", line 391, in __call__
I:     return self.run(*args, **kwds)
I:   File "/data/user/0/org.tribler.android/files/lib/python27.zip/unittest/case.py", line 327, in run
I:     testMethod()
I:   File "/data/data/org.tribler.android/files/lib/python2.7/site-packages/Tribler/Test/test_as_server.py", line 73, in check
I:     result = fun(*argv, **kwargs)
I:   File "/data/data/org.tribler.android/files/lib/python2.7/site-packages/Tribler/Test/Community/Multichain/test_multichain_community.py", line 170, in test_on_tunnel_remove_NoneType
I:     node.call(node.community.on_tunnel_remove, None, None, None, None, None)
I:   File "/data/data/org.tribler.android/files/lib/python2.7/site-packages/Tribler/dispersy/tests/debugcommunity/node.py", line 404, in call
I:     return blockingCallFromThread(reactor, func, *args, **kargs)
I:   File "/data/data/org.tribler.android/files/lib/python2.7/site-packages/Tribler/dispersy/util.py", line 236, in blockingCallFromThread
I:     this_thread_tb = traceback.extract_stack()
I:  Traceback from the reactor's thread:
I:    File "/data/data/org.tribler.android/files/lib/python2.7/site-packages/twisted/internet/defer.py", line 150, in maybeDeferred
I:     result = f(*args, **kw)
I:   File "/data/data/org.tribler.android/files/lib/python2.7/site-packages/Tribler/dispersy/util.py", line 37, in helper
I:     return func(*args, **kargs)
I: ok
```

--- a/Tribler/Test/Community/Multichain/test_multichain_community.py
+++ b/Tribler/Test/Community/Multichain/test_multichain_community.py
@@ -1,7 +1,10 @@
 """
 This file contains the tests for the community.py for MultiChain community.
 """
-import time
+from unittest.case import skip
+
+from nose.tools import raises
+
 from Tribler.Core.Session import Session
 from Tribler.community.multichain.community import (MultiChainCommunity, MultiChainCommunityCrawler, CRAWL_REQUEST,
                                                     CRAWL_RESPONSE, CRAWL_RESUME)
@@ -158,6 +161,8 @@ class TestMultiChainCommunity(AbstractServer, DispersyTestFunc):
         self.assertEqual((up, down), node.call(node.community._get_next_total, 0, 0))
         self.assertEqual((down, up), other.call(other.community._get_next_total, 0, 0))
 
+    @skip("Skip tunnel test that does not make sense")
+    @raises(AttributeError)
     def test_on_tunnel_remove_NoneType(self):
         """
         Test the on_tunnel_remove handler function to handle a NoneType

--- a/Tribler/Test/Community/Multichain/test_multichain_community.py
+++ b/Tribler/Test/Community/Multichain/test_multichain_community.py
@@ -167,7 +167,7 @@ class TestMultiChainCommunity(AbstractServer, DispersyTestFunc):
         other.send_identity(node)
         # Act
         try:
-            node.call(node.community.on_tunnel_remove, None, None, None, None, None)
+            node.call(node.community.on_tunnel_remove, None, None, None, None)
         except TypeError:
             error = True
         # Assert

